spicy/lib/spicy.spicy

```diff
@@ -80,4 +80,7 @@ public function current_time() : time &cxxname="::hilti::rt::time::current_time"
 public function bytes_to_hexstring(value: bytes) : string &cxxname="::spicy::rt::bytes_to_hexstring";
 
 ## Returns the value of an environment variable, if set.
-public function getenv(name: string) : optional<string> &cxxname="spicy::rt::getenv";
+public function getenv(name: string) : optional<string> &cxxname="::spicy::rt::getenv";
+
+## Formats a timestamp.
```

Comment:
This is the text that will make it into the documentation, would be good to expand a bit. I realize the other functions in this file don't have much info either, but this one would benefit a least from saying how a format string looks like (similar to the C++ API docs: POSIX style, with link to a page with reference of all format options).

Reply:
I expanded the comment here, but it does not look like `autogen-spicy-lib` pics up multiline docstrings. I filed #315 for that.